specification/postgresql/resource-manager/readme.md

```diff
@@ -60,6 +60,17 @@ input-file:
   - Microsoft.DBforPostgreSQL/stable/2022-12-01/ServerStartStopRestart.json
   - Microsoft.DBforPostgreSQL/stable/2022-12-01/VirtualNetwork.json
 ```
+
+### Tag: package-flexibleserver-2022-05-01-preview
+
+These settings apply only when `--tag=package-flexibleserver-2022-05-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-flexibleserver-2022-05-01-preview'
+input-file:
+  - Microsoft.DBforPostgreSQL/preview/2022-05-01-preview/Migrations.json
+  - Microsoft.DBforPostgreSQL/preview/2022-05-01-preview/Operations.json
+```
```

Comment:
This seems to be an "package-flexibleserver-2022-05-01-preview-only" tag?

For a complete tag, it should include everything in stable (except those upgraded for preview)

Reply:
Just as @weidongxu-microsoft said, if you want to release a beta sdk for these new swagger, please include everything in stable and with these 2 preview files.

Reply:
Change the tag name to `package-flexibleserver-2022-05-01-preview-only` if this only contains 2 files

Reply:
Changed as we don't require SDK for public preview